SeaPublicWebsite/SharedResources.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace SeaPublicWebsite;
+
+public class SharedResources
```

Comment:
Is this needed?

Reply:
Yes, this empty class is required according to Microsoft's documentation